src/webapp/components/landing-page-edit-dialog/LandingPageEditDialog.tsx

```diff
@@ -111,11 +114,40 @@ export const LandingPageEditDialog: React.FC<LandingPageEditDialogProps> = props
     };
 
     const handleFileUpload = useCallback(
-        (event: ChangeEvent<HTMLInputElement>) => {
+        (event: ChangeEvent<HTMLInputElement>, fileType: keyof LandingNode) => {
             const file = event.target.files ? event.target.files[0] : undefined;
+
             file?.arrayBuffer().then(async data => {
+                const reader = new FileReader();
+                reader.onload = e => {
+                    const img = new Image();
+                    img.onload = () => {
+                        const width = img.width;
+                        const height = img.height;
+                        const aspectRatio = width / height;
+
+                        setWarnings([]);
+                        const newWarnings: string[] = [];
+
+                        if (fileType === "favicon") {
+                            if (aspectRatio !== FAVICON_ASPECT_RATIO) {
```

Comment:
Although it is unfortunate, we have some client dashboards in which there is a pixel difference like 400x401 🥴. We could leave a margin of error of a maximum of 5 pixels or something like that. For example a error margin of `~0.015`